swedbank-pay-design-guide-jekyll-theme.gemspec

```diff
@@ -1,22 +1,30 @@
 # frozen_string_literal: true
 
+gem_version = "0.0.1.gemversionnotset"
+
+if ENV.has_key?("GEM_VERSION")
```

Comment:
I think it would be better to source the version number from a `version.json` file that is bundled with the gem somehow. We can create the `version.json` file in the build.